js/order.js

```diff
@@ -0,0 +1,12 @@
+export const OrderAPI = superclass =>
+    class extends superclass {
+        async getOrderTemplate(event) {

```

Comment:
If this is a templating API client then this can only be `getOrder`, otherwise we'll have `getXTemplate` everywhere.